packages/pyright-internal/jest.config.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
             },
         ],
     },
+    testTimeout: 6000,
```

Comment:
Was this an intentional change?

Reply:
This is from the package update change, @bschnurr might know?